### PR TITLE
"Flickering Stars" animation by RaHasib

### PR DESCRIPTION
**Pull request by RaHasib**
Thank you for making a pull request to ZTM's Animation Nation project. We will review your pull request as soon as we can, but keep in mind we can get very busy during Hacktoberfest.

While you're waiting, please look over this checklist. Feel free to write a description of what you did as well!

## What I did

*flickering Stars*

## Pull request checklist

- [x] This pull request uses **HTML and CSS** only and provides a `meta.json` file.
- [x] All files in this pull request are in a directory named: `<github_username>-<art_name>` without the <>.
- [x] Your pull request contains `index.html` and `styles.css`.
	- [x] Your html file is named exactly: `index.html`
	- [x] Your css file is named exactly: `styles.css`
- [x] Your pull request your `meta.json` with:
	- [x] `githubHandle`: being your unique github user name,
	- [x] `artName` : name of your animation
- [x] Your submission includes **at least one animation**.
- [x] Your pull request **does not modify any other files** in the repository.

#### Not normally needed
- [x] Check this if you would like to discuss an improvement besides adding animations

**Note: if your pull request makes an improvement besides adding a CSS animation you will need to discuss this with the maintainers. You can help kickstart this process by explaining what your change does and why you are making it in the "What I did" section**

## What to expect

- If your pull request meets the checklist requirements it will be approved and merged by a maintainer
- If your pull request needs some changes. A maintainer will leave a comment requesting changes
- If your pull request significantly violates the criteria in the checklist it will be closed with feedback.

### CODE REVIEW

File: Art/RaHasib-flickeringStars/index.html
Comment (LaurelineP) [inline Art/RaHasib-flickeringStars/index.html:1]: For the next times, in real life contribution,
you need to have clear indentations (nested div should be indented here )

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha @RaHasib Thanks for your contribution.
I would highly recommend you to double check pull requests you can do.
You've currently pushed files and folder you probably did not wanted to
image
Do not hesitate to review your PR before submitting in case you need to clean the branch :) 
As you can see your PR adds extra folders and files others than the files mentioned in the checklist.
Please address the following so we can merge your PR
- [ ] remove the whole hidden folders `.idea` and its files

File: Art/RaHasib-flickeringStars/index.html
Comment (RaHasib) [reply Art/RaHasib-flickeringStars/index.html:1]: Thank you for the feedback. I understand the importance of clear indentations for readability and maintainability of the code. I will make sure to follow this in my future contributions.

**Comment by LaurelineP** [comment]
Aloha bis @RaHasib , 
Thanks for your reactivity !
All the feedbacks are applied so it is good to go 🙌
Happy coding journey!

**Pull request merged**